pipelines/respondent-home-ui.yml

```diff
@@ -101,6 +111,25 @@ jobs:
   plan:
   - get: respondent-home-ui-source
     trigger: true
+  - get: ras-deploy
+  - task: run-unit-tests
+    file: ras-deploy/tasks/python-unit-tests.yml
+    input_mapping: { repository-name: respondent-home-ui-source }
+    on_failure:
+      put: notify
+      params:
+          text: |
+            Latest space respondent-home-ui unit tests failed. See build:
+            $ATC_EXTERNAL_URL/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME
+  - put: create-redis
+    resource: cf-cli-resource-latest
+    params:
+      command: create-service
+      service: awselasticache-redis
```

Comment:
For latest space I think this value should be `elasticache-broker`.

Reply:
I think you're right, as the other pipelines seem to use the same for latest (and CI).